src/main/java/org/broadinstitute/hellbender/tools/exome/IntervalBackedExonCollection.java

```diff
@@ -368,11 +410,11 @@ private int extendSearchIndexForward(final GenomeLoc location, final int startIn
      * Looks for the first index in {@link #sortedIntervals} that has an overlap with the input {@code location}
      * starting at {@code startIndex} and assuming that the element at that index has an overlap with {@code location}.
      */
-    private int extendSearchIndexBackwards(final GenomeLoc location, final int startIndex) {
-        final ListIterator<GenomeLoc> it = sortedIntervals.listIterator(startIndex);
+    private int extendSearchIndexBackwards(final SimpleInterval location, final int startIndex) {
+        final ListIterator<SimpleInterval> it = sortedIntervals.listIterator(startIndex);
         while (it.hasPrevious()) {
-            final GenomeLoc previous = it.previous();
-            if (previous.isBefore(location)) {
+            final SimpleInterval previous = it.previous();
+            if (intervalOverlapComparator.compare(previous,location) < 0) {
```

Comment:
I think it would be clear to just have `previous.getEnd() < previous.getStart()`